jkube-kit/jkube-kit-quarkus/src/main/java/org/eclipse/jkube/quarkus/QuarkusUtils.java

```diff
@@ -45,9 +45,12 @@ public class QuarkusUtils {
   private static final String QUARKUS_SMALLRYE_HEALTH_ROOT_PATH = "quarkus.smallrye-health.root-path";
   private static final String QUARKUS_SMALLRYE_HEALTH_READINESS_PATH = "quarkus.smallrye-health.readiness-path";
   private static final String QUARKUS_SMALLRYE_HEALTH_LIVENESS_PATH = "quarkus.smallrye-health.liveness-path";
+  private static final String QUARKUS_SMALLRYE_HEALTH_STARTUP_PATH = "quarkus.smallrye-health.startup-path";
   private static final String QUARKUS_MAVEN_PLUGIN_ARTIFACTID = "quarkus-maven-plugin";
   private static final int QUARKUS_MAJOR_VERSION_SINCE_PATH_RESOLUTION_CHANGE = 1;
   private static final int QUARKUS_MINOR_VERSION_SINCE_PATH_RESOLUTION_CHANGE = 11;
+  private static final int QUARKUS_MAJOR_VERSION_SINCE_STARTUP_CHANGE = 2;
+  private static final int QUARKUS_MINOR_VERSION_SINCE_STARTUP_CHANGE = 1;
```

Comment:
Could you please provide information on why we're checking for Quarkus 2.1 instead of Quarkus 2.0

AFAIK Smallrye health was updated to 3.0 in this PR https://github.com/quarkusio/quarkus/pull/16333 which was released along with Quarkus 2.0


https://github.com/eclipse/jkube/pull/1524#issuecomment-1149658415

Reply:
Just tried again on a simple Quarkus project with SmallRYE dependency. These were my observations:

- On 2.0.0.Final `/q/health/started` gives 404
- On 2.1.0.Final `q/health/started` gives 200 OK

Reply:
It seems that there was some work done on Quarkus in between the Smallrye dependency update and the startup probe wiring.